server/controllers/request.js

```diff
@@ -0,0 +1,104 @@
+const Request = require("../models/Request");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+const mongoose = require("mongoose");
+
+// @route GET /request
+// @desc get all request by user
+// @access Private
+exports.getRequestByUser = asyncHandler(async (req, res, next) => {
+    const { id } = req.user; 
+    
+    const requests = await Request.find({ 'user._id': id })
+
+    if(requests.length === 0){
+        res.status(404)
+        throw new Error("No Request found for this user")
+    }
+
+    res.send(requests)
```

Comment:
Consider it a good practice to add the status code: `res.status(200).send(requests)`

Reply:
Thanks for the heads-up. I didn't add `res.status(200)` because I noticed that a status of 200 is always sent as the status whenever I successfully send back a response, be it a file, JSON object, string e.t.c, I've only explicitly declared status like `res.status(somevalue).send(something)` whenever I intend to set its status to something other than 200. Are there exceptions to this behavior? Any thoughts? :-)

Reply:
Hey @BerryGIT-ME , 200 is a default status code when a successful response has been sent. What Sunday has mentioned above is a 'nice to have' as the code is more consistent :)